Work around for a test that should not build on non-glibc systems.

Problem and proper fix is reported to:
https://bugzilla.gnome.org/show_bug.cgi?id=756078

diff --git a/tests/pixbuf-lowmem.c b/tests/pixbuf-lowmem.c
index ae71788..9e9571f 100644
--- a/tests/pixbuf-lowmem.c
+++ b/tests/pixbuf-lowmem.c
@@ -24,6 +24,7 @@
 #include <time.h>
 #include <string.h>
 
+#ifdef __GLIBC__
 #define PRETEND_MEM_SIZE (16 * 1024 * 1024)
 #define REMAINING_MEM_SIZE 100000
 
@@ -225,3 +226,10 @@ main (int argc, char **argv)
   
   return 0;
 }
+#else
+int
+main (int argc, char **argv)
+{
+	return 0;
+}
+#endif
